### PR TITLE
Create base project

### PR DESCRIPTION
**Pull request by jdhiguita**
## Summary
- Completed the project setup
- Finished PR structure
## Trello Card
https://trello.com/c/Fe0jjB6b

### CODE REVIEW

File: .gradle/buildOutputCleanup/cache.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#Wed Aug 19 09:04:08 COT 2020
+gradle.version=6.4.1
```
Comment (cramallo) [inline .gradle/buildOutputCleanup/cache.properties]: All .gradle should not be in the remote repository.

File: .idea/.name [hunk 2]
```diff
@@ -0,0 +1 @@
+training
```
Comment (cramallo) [inline .idea/.name]: All .idea should not be in the remote repository

**Review by cramallo — CHANGES_REQUESTED** [review]

File: .gradle/buildOutputCleanup/cache.properties (hunk 1 above)
Comment (cramallo) [reply .gradle/buildOutputCleanup/cache.properties]: add this to .gitignore

Comment (jdhiguita) [reply .gradle/buildOutputCleanup/cache.properties]: .gitignore has been added and it is now working

File: .idea/.name (hunk 2 above)
Comment (jdhiguita) [reply .idea/.name]: All .idea it is all out

**Review by cramallo — APPROVED** [review]

**Review by Gianni-Nicolas — APPROVED** [review]
Great Job 😄

**Pull request merged**